site/gatsby-site/server/tests/notifications.spec.ts

```diff
@@ -1540,4 +1540,138 @@ describe(`Notifications`, () => {
             },
         ]);
     });
+
+    it('Should not crash if no recipients found', async () => {
```

Comment:
It looks like this test is failing https://github.com/responsible-ai-collaborative/aiid/actions/runs/12437299433/job/34998005994?pr=3301